gradle.properties

```diff
@@ -0,0 +1,14 @@
+# Scan tool properties
+scanToolVersion=1.0.0
+
+# dependencies
+spotbugsPluginVersion=5.0.14
+downloadPluginVersion=5.4.0
+picoCLIVersion=4.7.5
+gsonVersion=2.10.1
+ballerinaVersion=2201.8.5
```

Comment:
AFAIK this should be `ballerinaLangVersion`. Check with stdlib repos and use the same way.

Reply:
Fixed in [5c9e9f8](https://github.com/ballerina-platform/static-code-analysis-tool/pull/2/commits/5c9e9f806ffb2fcb8658f63f7e99981c74ceed04)